Dockerfile

```diff
@@ -10,4 +10,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java $JAVA_OPTS -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar nakadi.jar
\ No newline at end of file
+ENTRYPOINT java $JAVA_OPTS $(appdynamics-agent) -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar \
```

Comment:
Oh, is there a way not to have appdinamics dependency like this in our open-source project?